nvme-metrics.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
```

Comment:
Do you provide all metrics that `nvme_metrics.sh` provides? If yes, please remove `nvme_metrics.sh`. This Python version is much cleaner and better in every regard.